corehq/apps/analytics/tasks.py

```diff
@@ -193,6 +193,13 @@ def track_user_sign_in_on_hubspot(webuser, cookies, meta, path):
 
 
 @analytics_task()
+def hubspot_track_user_phone_number_on_signup(webuser):
```

Comment:
you can probably just use `update_hubspot_properties` https://github.com/dimagi/commcare-hq/pull/12825/files#diff-f91ad376200614f22e56c169ab86b9a7R174, without needing to create a new function. However, that relies on the user having already been created. If its possible their data isn't in hubspot yet, it may be safer to include it in the dict I linked above, which is sent when the user account is created for all users. Since the new phone number property should be on all user docs, even if it is empty, it is probably safe to check for its existence there.
I also just checked hubspot and the existing phone number property `phone`, so its easiest to just use that as the dict key.
As a side note, im working on updating the analytics readme now, so all of these best practices will be documented.